### ISSUE TITLE
Documentation for gradient estimators

### ISSUE THREAD

**Issue by jeremiecoullon** [jeremiecoullon/SGMCMCJax#2]
Do we need documentation for the gradient estimators? In theory, no one should ever worry about them as they are at the "diffusion" level of abstraction. But perhaps this would be useful.

**Comment by jeremiecoullon** [comment]
This is now both in the [API docs](https://sgmcmcjax.readthedocs.io/en/latest/gradient_estimation.html) and in the ["all samplers"](https://sgmcmcjax.readthedocs.io/en/latest/all_samplers.html) section of the docs
